test/extended/operators/tolerations.go

```diff
@@ -31,7 +31,7 @@ var _ = Describe("[sig-arch] Managed cluster should", func() {
 		namespacePrefixes := sets.NewString("kube-", "openshift-")
 		excludedNamespaces := sets.NewString("openshift-kube-apiserver", "openshift-kube-controller-manager", "openshift-kube-scheduler", "openshift-etcd", "openshift-openstack-infra", "openshift-ovirt-infra")
 		// exclude these pods from checks
-		whitelistPods := sets.NewString("network-operator", "dns-operator", "olm-operators", "gcp-routes-controller", "ovnkube-master", "must-gather")
+		whitelistPods := sets.NewString("network-operator", "dns-operator", "olm-operators", "gcp-routes-controller", "ovnkube-master", "must-gather", "kube-apiserver-proxy")
```

Comment:
why add the pod name to the whitelist *and* add logic to generally exclude staticpods?

Reply:
That is wrong, I am happy to do either